prometheus-exporters/vrops-exporter/alerts/virtual_machine.alerts

```diff
@@ -0,0 +1,20 @@
+groups:
+- name: virtual_machine.alerts
+  rules:
+  - alert: InaccessibleShadowVM
+    expr: >
+      vrops_vm_stats{statkey="number_vcpus_total"} == 1
+      and on(virtualmachine) vrops_vm_stats{statkey="vm_memory_kilobytes"} == 131072
+      and on(virtualmachine) vrops_vm_properties{propkey="summary_ethernetCards"} == 0
+      and on(virtualmachine) vrops_vm_properties{propkey="runtime_connectionState: disconnected"} == 1
+    for: 5m
```

Comment:
do you want to fire straight? usually i am giving it 15-30mins to sit on a matched condition before firing.

Reply:
You are right. Increased to 20 min